web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -694,6 +694,19 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                 })}
               </Select>
             </Form.Item>
+            <Form.Item
+              label={formatMessage({ id: 'page.upstream.step.scheme' })}
+              name="scheme"
+            >
+              <Select disabled={readonly}>
+              {['http', 'https', 'grppc', 'grpcs'].map((item, index) => (
```

Comment:
Hi @batman-ezio , as refer to https://github.com/apache/apisix/blob/master/docs/en/latest/admin-api.md#upstream, I guess the third one `grppc` is a typo here.

Reply:
thanks. updated